docs/voicemail-how-to.md

```diff
@@ -55,15 +55,17 @@ Then you can add `voicemail-grpc` and `common`:
 import com.wgtwo.api.auth.Clients
 import com.wgtwo.api.common.OperatorToken
 
-val channel = Clients.createChannel(Clients.Environment.PROD)
-val credentials = OperatorToken("YOUR_CLIENT_ID", "YOUR_CLIENT_SECRET")
+val channel: ManagedChannel = Clients.createChannel(Environment.PROD)
+val credentials = OperatorToken(Secrets.WGTWO_CLIENT_ID, Secrets.WGTWO_CLIENT_SECRET)
 val blockingStub = VoicemailMediaServiceGrpc.newBlockingStub(channel).withCallCredentials(credentials)
 ```
 
 ## List voicemails
 ```kotlin
 fun listVoicemails(msisdn: String): MutableList<Voicemail.VoicemailMetadata>? {
-    val voicemailMetadataRequest = Voicemail.GetAllVoicemailMetadataRequest.newBuilder().setMsisdn(msisdn).build()
+    val phoneNumberProto = PhoneNumberProto.PhoneNumber.newBuilder().setE164(msisdn)
```

Comment:
Does this work? (since the msisdn doens't have the `+)

Reply:
yes, it works but only because on the server side we do `.toMsisdn()` on the PhoneNumber. I updated the snippets code though to specify the format (this PR: https://github.com/working-group-two/wgtwo-kotlin-code-snippets/pull/4/files).